tests/conftest.py

```diff
@@ -28,3 +31,9 @@ def _load(filename):
         return base_path / filename
 
     return _load
+
+
+@pytest.fixture
+def do_minimal():
```

Comment:
Setup a reusable `do` object under `conftest.py` Calling it `do_minimal` since it has the minimal number of arguments and setup of `xarray`, `yarray`, `xtype` and `wavelength`

Reply:
I like this name. I suggested test_do or something, but your name is better

Reply:
done